pyface/tests/test_message_dialog.py

```diff
@@ -130,6 +130,13 @@ def test_error(self):
         with self.event_loop():
             self.dialog.destroy()
 
+    @unittest.skipIf(
+        is_pyqt5, "Message dialog click tests don't work on pyqt5."
+    )
+    @unittest.skipIf(
+        is_pyqt4_linux,
```

Comment:
Has this been tested with the new PyQt 4.12.1 eggs? Those seem to have resolved some of the stability issues on Linux.

Reply:
Just tried this out of curiosity - it it segfaults.